agent/structs/acl.go

```diff
@@ -75,11 +77,58 @@ session_prefix "" {
 	policy = "write"
 }` + EnterpriseACLPolicyGlobalManagement
 
-	ACLReservedPrefix = "00000000-0000-0000-0000-0000000000"
+	ACLPolicyGlobalReadOnlyID    = "00000000-0000-0000-0000-000000000002"
+	ACLPolicyGlobalReadOnlyName  = "builtin/global-read-only"
+	ACLPolicyGlobalReadOnlyDesc  = "Builtin Policy that grants unlimited read-only access to all components"
+	ACLPolicyGlobalReadOnlyRules = `
+acl = "read"
+agent_prefix "" {
+	policy = "read"
+}
+event_prefix "" {
+	policy = "read"
+}
+key_prefix "" {
+	policy = "read"
+}
+keyring = "read"
+node_prefix "" {
+	policy = "read"
+}
+operator = "read"
+mesh = "read"
+peering = "read"
+query_prefix "" {
+	policy = "read"
+}
+service_prefix "" {
+	policy = "read"
+	intentions = "read"
+}
+session_prefix "" {
+	policy = "read"
```

Comment:
What are your thoughts on abstracting these rules out into a shared template? I think it could make maintaining the rules easier going forward. e.g:
```
aclPolicyGlobalRulesTemplate = `
acl = "%s"
agent_prefix "" {
	policy = "%s"
}
...
`

ACLPolicyGlobalReadOnlyRules = fmt.Sprintf(aclPolicyGlobalRulesTemplate, "read")
ACLPolicyGlobalManagementRules = fmt.Sprintf(aclPolicyGlobalRulesTemplate, "write")
```